lib/srv/desktop/windows_server.go

```diff
@@ -493,6 +527,10 @@ func (s *WindowsService) tlsConfigForLDAP() (*tls.Config, error) {
 func (s *WindowsService) Close() error {
 	s.close()
 
+	if s.sidCache != nil {
+		s.sidCache.Shutdown(s.closeCtx)
+	}
```

Comment:
Is this necessary? The `FnCache` is running with the service's context, which was closed a couple lines above.